pulpcore/tests/functional/__init__.py

```diff
@@ -95,6 +95,20 @@ def pytest_configure(config):
 # API Clients
 
 
+@pytest.fixture(scope="session")
+def bindings_cfg():
+    api_protocol = os.environ.get("API_PROTOCOL", "https")
+    api_host = os.environ.get("API_HOST", "localhost")
+    api_port = os.environ.get("API_PORT", "80")
```

Comment:
Maybe, changing the port value would be better?
```suggestion
    api_port = os.environ.get("API_PORT", "443")
```

```
cat .config/pulp_smash/seetings.json

{
  "hosts": [
    {
      "hostname": "pulp3-source-fedora35.localhost.example.com",
      "roles": {
        "api": {
          "port": 443,
          "scheme": "https",
          "service": "nginx",
          "verify": false
        },
        "content": {
          "port": 443,
          "scheme": "https",
          "service": "pulpcore-content",
          "verify": false
        },
        "pulp resource manager": {},
        "pulp workers": {},
        "redis": {},
        "shell": {
          "transport": "local"
        }
      }
    }
  ],
  "pulp": {
    "auth": [
      "admin",
      "password"
    ],
    "selinux enabled": false,
    "version": "3"
  }
}
```

Reply:
Yes, the variables are set by oci_env. And, I think we should set the variables in the CI too. Any objections, @dkliban?

Note that a vagrant installation has no notion about these variables. We need to set the variables in the installer if we plan to support it.